atom/browser/web_dialog_helper.cc

```diff
@@ -117,9 +117,10 @@ class FileSelectHelper : public base::RefCounted<FileSelectHelper>,
           base_dir = paths[0];
 
           // Actually enumerate soemwhere off-thread
-          base::SequencedTaskRunnerHandle::Get()->PostTask(
-              FROM_HERE, base::BindOnce(&FileSelectHelper::EnumerateDirectory,
-                                        this, base_dir));
+          base::PostTaskWithTraits(
+              FROM_HERE, {content::BrowserThread::UI},
```

Comment:
The original intent was to post the task to a different sequence, `content::BrowserThread::UI` should be avoided.